### ISSUE TITLE
Call to undefined method PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocNode::getRequireExtendsTagValues()

### ISSUE THREAD

**Issue by jonw-cogapp** [craftcms/rector#18]
### Description

As a dependency of Rector, PHPStan 1.10.55 created [a recent issue with rector 0.18.x](https://github.com/rectorphp/rector/issues/8395), which was resolved in rector 0.19.0. However `craftcms/rector` requires rector ^0.18.4, so it seems there isn't an upgrade pathway to mitigate this error until `craftcms/rector` supports rector 0.19.x.

```
 [ERROR] Could not process "config/app.php" file, due to:                                                               
         "Call to undefined method PHPStan\PhpDocParser\Ast\PhpDoc\PhpDocNode::getRequireExtendsTagValues()". On line:  
         330
```

### Steps to reproduce

1. `composer require --dev rector/rector:^0.18`
2. [Follow the instructions in the readme for including craftcms/rector](https://github.com/craftcms/rector) 
3. [Add a `rector.php` file using the given example](https://github.com/craftcms/rector?tab=readme-ov-file#advanced-configuration)
4. `vendor/bin/rector --dry-run`

### Additional info

- Craft version: 4.6
- PHP version: 8.2

**Comment by khalwat** [comment]
Ran into this today...

**Comment by khalwat** [comment]
Resolved via PR ([#19](https://github.com/craftcms/rector/pull/19))
